### ISSUE TITLE
Wind speed metric

### ISSUE THREAD

**Issue by gideonite** [ai2cm/ace#247]
[GenCast](https://arxiv.org/abs/2312.15796) makes a strong argument for computing and comparing wind speed. I think this should be implemented as an additional metric.

![image](https://github.com/ai2cm/ace/assets/2163686/9782ade9-f4cb-4b1d-9815-d5f08ac6cb4d)

**Comment by oliverwm1** [comment]
Yes, this is a pretty standard diagnostic to output from climate models and I agree would be useful to track. I would suggest just computing it for the lowest vertical layer since this is where wind speed is particularly critical w.r.t. accurately estimating surface turbulent fluxes and coupling to land/ocean.

**Comment by gideonite** [comment]
Great idea. Assigning this to myself. Will aim to get it done by the end of the week.
